Gemfile.lock

```diff
@@ -132,6 +132,8 @@ GEM
     nio4r (2.5.8)
     nokogiri (1.13.6-x64-mingw-ucrt)
       racc (~> 1.4)
+    nokogiri (1.13.6-x86_64-darwin)
```

Comment:
This is me running bundle install for mac

Reply:
Didn't consider this possibility. Thanks for making the inclusion.